ui/src/cluster-workflow-templates/cluster-workflow-template-markdown.tsx

```diff
@@ -13,10 +14,13 @@ interface ClusterWorkflowTemplateMarkdownProps {
 export function ClusterWorkflowTemplateMarkdown(props: ClusterWorkflowTemplateMarkdownProps) {
     const wf = props.workflow;
     // title + description vars
-    const title = wf.metadata.annotations?.[ANNOTATION_TITLE] ?? wf.metadata.name;
-    const description = (wf.metadata.annotations?.[ANNOTATION_DESCRIPTION] && `\n${wf.metadata.annotations[ANNOTATION_DESCRIPTION]}`) || '';
-    const hasAnnotation = title !== wf.metadata.name || description !== '';
+    const title = (wf.metadata.annotations?.[ANNOTATION_TITLE] && `${escapeInvalidMarkdown(wf.metadata.annotations[ANNOTATION_TITLE])}`) ?? wf.metadata.name;
```

Comment:
Why using `??` instead of `||` here ? You have a chance of having an empty title

Reply:
Thanks for the feedback, @Adrien-D! The syntax for the `title` was taken from an [existing line](https://github.com/argoproj/argo-workflows/blob/10aaf3eb1873d88d3f9ab3027d9452b40c91d9d9/ui/src/workflows/components/workflows-row/workflows-row.tsx#L33) for setting the title in the `workflows-row.tsx` file. Seems the use of the nullish coalescing operator is to ensure users can enter values that would otherwise be evaluated as "false-y." This approach was carried over to ensure that defaulting to the `wf.metadata.name` is only in the case of a `null` or `undefined` value in the `[ANNOTATION_TITLE]` field.

Please let me know if the use of the `||` operator is a preferred method to use here and I can go through the files and make the update.

Reply:
There is not really a preferred method, but as it's taken from an existing line, it's all good !